app/_gateway_entities/route.md

```diff
@@ -4,57 +4,172 @@ content_type: reference
+
+### Path matching
+
+Keep the following path matching recommendations in mind when configuring paths:
+
+* **Regex in paths:** For a path to be considered a regular expression, it must be prefixed with a `~`. You can avoid creating complex regular expressions using the [Router Expressions language](/gateway/routing/expressions/).
+* **Capturing groups:** Capturing groups are also supported, and the matched group will be extracted from the path and available for plugins consumption.
+* **Escaping special characters:** When configuring Routes with regex paths via the Admin API, be sure to URL encode your payload if necessary according to [RFC 3986](https://tools.ietf.org/html/rfc3986).
+* **Normalization behavior:** To prevent trivial Route match bypass, the incoming request URI from client
+is always normalized according to [RFC 3986](https://tools.ietf.org/html/rfc3986)
+before router matching occurs.
+
+  Regex Route paths only use methods 1 and 2. In addition, if the decoded character becomes a regex meta character, it will be escaped with backslash.
+
+### Priority matching
+
+If multiple Routes match, the {{site.base_gateway}} router then orders all defined Routes by their priority and uses the highest priority matching Route to handle a request. How Routes are prioritized depends on the router mode you're using.
+
+#### Traditional compatibility mode
+
+In `traditional_compat` mode, the priority of a Route is determined as
+follows, by the order of descending significance:
+
+1. **Priority points:** For the presence of each of a Route's `methods`, `host`, `headers`, and `snis`, a "priority point" is added. Routes with higher "priority point" values will be considered before those with lower values.
+2. **Wildcard hosts:** Among Routes with the same "priority point" value, those that have any wildcard host specification will be considered after those that don't have any wildcard host (or no host) specified.
+3. **Header count:** The resulting groups are sorted so the Routes with a higher number of specified headers have higher priority than those with a lower number of headers.
+4. **Regular expressions and prefix paths:** Routes that have a regular expression path are considered first and are ordered by their `regex_priority` value. Routes that have no regular expression path are ordered by the length of their paths. 
+
+When two Routes have the same path, {{site.base_gateway}} uses a tiebreaker. For example, if the rule count for the given request is the same in two Routes `A` and `B`, then the following tiebreaker rules will be applied in the order they are listed. Route `A` will be selected over `B` if:
+  * `A` has only "plain" Host headers and `B` has one or more "wildcard"
+  host headers
+  * `A` has more non-Host headers than `B`.
+  * `A` has at least one "regex" paths and `B` has only "plain" paths.
+  * `A`'s longest path is longer than `B`'s longest path.
+  * `A.created_at < B.created_at`
```

Comment:
The `created_at` rule is missing from the list before this. Looks like that's the final priority rule, aka rule number 5? If all of the above are  equal, the router chooses the route that was created first.